app/src/main/java/com/example/android/sunshine/app/sync/SunshineSyncAdapter.java

```diff
@@ -95,7 +95,11 @@ public SunshineSyncAdapter(Context context, boolean autoInitialize) {
     @Override
     public void onPerformSync(Account account, Bundle extras, String authority, ContentProviderClient provider, SyncResult syncResult) {
         Log.d(LOG_TAG, "Starting sync");
-        String locationQuery = PrefUtility.getPreferredLocation(getContext());
+
+        Context context = getContext();
+        String locationQuery = PrefUtility.getPreferredLocation(context);
+        String locationLatitude = String.valueOf(PrefUtility.getLocationLatitude(context));
```

Comment:
could you have `Latitude` and `Longitude` value types? if you pass one of them to `appendQueryParameter(` does it take `Object` (and so call `toString internally)?